homeassistant/components/alexa/smart_home.py

```diff
@@ -38,13 +38,12 @@
 }
 
 API_THERMOSTAT_MODES = {
-    climate.STATE_HEAT: 'HEAT',
-    climate.STATE_COOL: 'COOL',
-    climate.STATE_AUTO: 'AUTO',
-    climate.STATE_ECO: 'ECO',
-    climate.STATE_IDLE: 'OFF',
-    climate.STATE_FAN_ONLY: 'OFF',
-    climate.STATE_DRY: 'OFF',
+    'HEAT': [climate.STATE_HEAT],
```

Comment:
The old code has a bug with `OFF` mode.   When I said 'Alex, set to <device name> to OFF', it didn't turn off my A/C.  Instead, it set my A/C to `dry` mode because `dry` is mapped to `OFF` too.

Reply:
Guess you're running Python 3.5? If `API_THERMOSTAT_MODES` was an `OrderedDict` (as dicts are by default in 3.6+), this won't happen. This PR too complicated, revert the change to `API_THERMOSTAT_MODES` and add a test to make sure that off is returned instead of dry.

Reply:
Thanks for the hint!
Let me try it